tests/js/spec/components/sidebar/__snapshots__/index.spec.jsx.snap

```diff
@@ -989,75 +974,55 @@ exports[`Sidebar SidebarHelp can toggle help menu 1`] = `
     </SidebarMenuItem>
     <SidebarMenuItem
       href="https://forum.sentry.io/"
-      openInNewTab={true}
     >
       <StyledSidebarMenuItemLink
         href="https://forum.sentry.io/"
-        openInNewTab={true}
       >
         <SidebarMenuItemLink
-          className="css-1eunyni-StyledSidebarMenuItemLink eev51g21"
+          className="css-1sehl9l-StyledSidebarMenuItemLink eev51g21"
           href="https://forum.sentry.io/"
-          openInNewTab={true}
         >
-          <Link
-            className="css-1eunyni-StyledSidebarMenuItemLink eev51g21"
-            href="https://forum.sentry.io/"
-            target="_blank"
+          <div
+            className="css-1sehl9l-StyledSidebarMenuItemLink eev51g21"
+            tabIndex={0}
           >
-            <a
-              className="css-1eunyni-StyledSidebarMenuItemLink eev51g21"
-              href="https://forum.sentry.io/"
-              target="_blank"
+            <MenuItemLabel
+              hasMenu={true}
             >
-              <MenuItemLabel
-                hasMenu={false}
+              <span
+                className="css-uvye8q-MenuItemLabel-MenuItemLabel eev51g20"
               >
-                <span
-                  className="css-1ug1il4-MenuItemLabel-MenuItemLabel eev51g20"
-                >
-                  Community Discussions
-                </span>
-              </MenuItemLabel>
-            </a>
-          </Link>
+                Community Discussions
+              </span>
+            </MenuItemLabel>
+          </div>
```

Comment:
Hmm is this right? I would expect to see `<ExternalLink>` somewhere.

Reply:
good catch..return was missing ;-)